tests/mypy/outputs/plugin-fail-strict.txt

```diff
@@ -1,34 +1,35 @@
-23: error: Unexpected keyword argument "z" for "Model"  [call-arg]
```

Comment:
We need to come up with a better solution for these diffs :/

Reply:
Sorry, I could not understand why we should change it.
Extra argument is not allowed here by `config`. Is it a problem? :thinking:

Reply:
Sorry, this wasn't directed at you, or specific to this PR -- your changes are fine. It would just be good if the diff was easier to make sense of, or if we could more easily test one thing at a time.

Something to fix in a future PR, not this one. 👍